saleor/product/migrations/tasks/saleor3_9.py

```diff
@@ -17,14 +17,14 @@ def set_description_plaintext(qs: QuerySet[Product]):
 
 
 @app.task
-def set_description_plaintext_task():
+def set_description_plaintext_task(last_pk: int = 0):
```

Comment:
Don't we need to change the direction of ordering if we start from 0 and use `greater_than`?